code/game/machinery/camera/tracking.dm

```diff
@@ -108,53 +114,37 @@
 
 /mob/living/silicon/ai/proc/trackable_mobs()
 
+	track.names.Cut()
+	track.namecounts.Cut()
+	track.humans.Cut()
+	track.others.Cut()
+
 	if(usr.stat == 2)
 		return list()
 
-	var/datum/trackable/TB = new()
 	for(var/mob/living/M in mob_list)
-		// Easy checks first.
-		// Don't detect mobs on Centcom. Since the wizard den is on Centcomm, we only need this.
-		if(InvalidTurf(get_turf(M)))
-			continue
-		if(M == usr)
-			continue
-		if(M.invisibility)//cloaked
-			continue
-		if(M.digitalcamo)
+		if(!M.can_track(usr))
 			continue
 
 		// Human check
 		var/human = 0
 		if(istype(M, /mob/living/carbon/human))

```

Comment:
well, it was more uh, monkies being carbons on TG and being excluded. It's not a big deal, was just thinking about it, s'all.